content/build/building/best-practices.md

```diff
@@ -1,11 +1,255 @@
 ---
-description: Hints, tips and guidelines for writing clean, reliable Dockerfile instructions
-keywords: parent image, images, dockerfile, best practices, hub, official image
-title: Best practices for Dockerfile instructions 
+description: Hints, tips and guidelines for writing clean, reliable Dockerfiles
+keywords: base images, dockerfile, best practices, hub, official image
+title: Image-building best practices
```

Comment:
Not sure about "Image-building", I think "Building" is more appropriate. User might just want to build and output binaries for example.